src/en/news/blog/2023/v18.2.0-released/index.md

```diff
@@ -0,0 +1,523 @@
+  completing the RGW upgrades before creating or modifying any notification
+  topics.
+
+- Compression is now supported for objects uploaded with Server-Side
+  Encryption. When both compression and encryption are enabled, compression is
+  applied before encryption. Earlier releases of multisite do not replicate
+  such objects correctly, so all zones must upgrade to Reef before enabling the
+  `compress-encrypted` zonegroup feature: see
+  https://docs.ceph.com/en/reef/radosgw/multisite/#zone-features and note the
+  security considerations.
+
+### Unsorted
+
+- Telemetry: Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard. For more details, see:
+  https://docs.ceph.com/en/reef/mgr/telemetry/#leaderboard. To see a sample
+  report, run ``ceph telemetry preview``. To opt in to telemetry, run ``ceph
+  telemetry on``. To opt in to the leaderboard, run ``ceph config set mgr
+  mgr/telemetry/leaderboard true``. To add a leaderboard description, run
+  ``ceph config set mgr mgr/telemetry/leaderboard_description ‘Cluster
+  description’`` (entering your own cluster description).
+
+## <a id="upgrade"></a>Upgrading from Pacific or Quincy
+
+Before starting, make sure your cluster is stable and healthy (no down or recovering OSDs). (This is optional, but recommended.) You can disable the autoscaler for all pools during the upgrade using the noautoscale flag.
+
+**Note**: You can monitor the progress of your upgrade at each stage with the `ceph versions` command, which will tell you what ceph version(s) are running for each type of daemon.
+
+### Upgrading cephadm clusters
+
+If your cluster is deployed with cephadm (first introduced in Octopus), then the upgrade process is entirely automated. To initiate the upgrade,
+
+    ceph orch upgrade start --ceph-version 18.2.0
+
+The same process is used to upgrade to future minor releases.
+
+Upgrade progress can be monitored with `ceph -s` (which provides a simple progress bar) or more verbosely with
+
+    ceph -W cephadm
+
+The upgrade can be paused or resumed with
+
+    ceph orch upgrade pause  # to pause
+    ceph orch upgrade resume # to resume
+
+or canceled with
+
+    ceph orch upgrade stop
+
+Note that canceling the upgrade simply stops the process; there is no ability to downgrade back to Pacific or Quincy.
+
+### Upgrading non-cephadm clusters
+
+**Note**: If you cluster is running Octopus (15.2.x) or later, you might choose to first convert it to use cephadm so that the upgrade to Quincy is automated (see above). For more information, see https://docs.ceph.com/en/reef/cephadm/adoption/.
+
+1. Set the `noout` flag for the duration of the upgrade. (Optional, but recommended.)
+
+   ceph osd set noout
```

Comment:
Hmm, to few spaces? In contrast to the snippets directly above, GH doesn't color this one and following. I don't know whether this has any impact on final document generated from this source.

Reply:
Will fix this